src/NUnitTestExtractor/NUnitTestExtractor.Tests/NUnitTestExtractorTests.cs

```diff
@@ -3,22 +3,309 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.IO;
 using NUnit.Framework;
+using NUnitTestExtractor;
+using System.Diagnostics;
+using System.Collections;
 
-namespace NUnitTestExtractor.Tests
+namespace NUnitTestExtractorTests
 {
     [TestFixture]
     public class NUnitTestExtractorTests
     {
-        [Test]
-        public void DirectoryCreatedWhenOutput()
+        private const string outputDirectory = "C:/Git/NTestController/src/NUnitTestExtractor/NUnitTestExtractor.Tests/bin/Debug/TestOutput/file.txt";
```

Comment:
Never hardcode absolute paths like this, otherwise the tests won't be able to run on every machine since a lot of people won't have a C:\Git directory, and if they're running Linux, they won't even have a C:\
You should use relative paths based on your current directory instead.
Also, the name says outputDirectory, but it's actually the full path and filename.